src/welcome/index.js

```diff
@@ -6,6 +6,7 @@ const github = require('@actions/github');
     const githubToken = core.getInput('github-token', { required: true });
     const issueMessage = core.getInput('issue-message');
     const prMessage = core.getInput('pr-message');
+    const prFooter = core.getInput('footer');
```

Comment:
The footer message is not just for the PR it is also for the Issue too, so I think `footer` for the variable name is better

Reply:
Okay, It is now clear